content/applications/productivity/iot/config/updating_iot.rst

```diff
@@ -0,0 +1,140 @@
+
+Then, download the latest :abbr:`IoT (Internet of Things)` image from `nightly
+<http://nightly.odoo.com/master/iotbox>`_, which will be labeled as `iotbox-latest.zip`. This
+particular image is compatible with *all* supported versions of Odoo.
+
+After this step is complete, insert the :abbr:`IoT (Internet of Things)` box's micro SD card into
+the computer or reader. Open *Etcher*, and select :guilabel:`Flash from file`, then find and select
+the `iotbox-latest.zip` image and extract it. Next, select the drive the image should be burned to.
+
+Lastly, click :guilabel:`Flash`, and wait for the process to finish.
+
+.. image:: updating_iot/etcher-app.png
+   :align: center
+   :alt: Balena's Etcher software dashboard.
+
+.. tip::
+   Balena's *Etcher* software also allows for the administrator to flash the :abbr:`SD (Secure
+   Digital)` card from a :abbr:`URL (Uniform Resource Locator)`. To flash from a :abbr:`URL (Uniform
+   Resource Locator)`, simply click :guilabel:`Flash from URL`, instead of :guilabel:`Flash from
+   file`.
+
+   Then, enter the following: `http://nightly.odoo.com/master/iotbox/iotbox-latest.zip`.
+
+   .. image:: updating_iot/url-flash.png
+      :align: center
+      :alt:  A view of Balena's Etcher software, with the flash from URL option highlighted.
+
+.. note::
+   An alternative software for flashing the micro SD card is *Raspberry Pi Imager*. Download the
+   *Raspberry Pi* software `here <https://www.raspberrypi.com/software/>`_.
+
+.. _iot/config/homepage-upgrade:
+
+Update from the IoT box home page
+=================================
+
+In the background, the :abbr:`IoT (Internet of Things)` box uses a version of Odoo code to run and
+connect to the Odoo database. This code may need to be updated in order for the :abbr:`IoT (Internet
+of Things)` box to operate effectively. This operation should be completed on a routine basis, to
+ensure the :abbr:`IoT (Internet of Things)` system, and its processes, stay up-to-date.
+
+Go to the :abbr:`IoT (Internet of Things)` box homepage by navigating to :menuselection:`IoT app -->
+IoT Boxes`, and clicking on the :guilabel:`IP address` of the :abbr:`IoT (Internet of Things)` box.
+Then, click on :guilabel:`Update` (next to the version number).
+
+If a new version of the :abbr:`IoT (Internet of Things)` box image is available, an
+:guilabel:`Upgrade to _xx.xx_` button appears at the bottom of the page. Click this button to
+upgrade the unit, at which point the :abbr:`IoT (Internet of Things)` box flashes itself to the
+newer version. All of the previous configurations are then saved.
+
+.. note::
+   This process can take more than 30 minutes. Do **not** turn off, or unplug, the :abbr:`IoT
+   (Internet of Things)` box, as it would leave it in an inconsistent state. This means the
+   :abbr:`IoT (Internet of Things)` box needs to be :ref:`re-flashed <iot/config/flash>` with a new
+   image.
+
+   .. seealso::
+      :ref:`iot/config/flash`.
```

Comment:
Suggestion to make this an important admonition and can remove the seealso since there is an inline reference used in the last sentence to the same location
```suggestion
.. important::
   This process can take more than 30 minutes. Do **not** turn off, or unplug, the :abbr:`IoT
   (Internet of Things)` box, as it would leave it in an inconsistent state. This means the
   :abbr:`IoT (Internet of Things)` box needs to be :ref:`re-flashed <iot/config/flash>` with a new
   image.
```